frontend/src/app/account-activation/account-activation.component.ts

```diff
@@ -0,0 +1,40 @@
+import { Component, OnInit } from '@angular/core';
+import { AccountActivationRequestModel } from 'app/auth.models';
+import { Observable } from 'rxjs/Observable';
+import { AuthService } from '../auth.service';
+import { Router, ActivatedRoute, Params } from '@angular/router';
+
+@Component({
+  selector: 'volontulo-account-activation',
+  templateUrl: './account-activation.component.html',
+  styleUrls: ['./account-activation.component.css']
+})
+export class AccountActivationComponent implements OnInit {
+  accountActivationRequestModel: AccountActivationRequestModel = {
+    uuid: '',
+  };
+  activationSuccessful: boolean;
+
+  constructor(private authService: AuthService, private activatedRoute: ActivatedRoute
+  ) {
+    this.activationSuccessful = false;
```

Comment:
AFAIK You can do that when You define attribute in class (TS will move that to automatically created constructor)